src/Controller/ArticleController.php

```diff
@@ -25,11 +26,30 @@ public function listArticle(Request $request, ArticleRepository $articleReposito
         $page = $request->query->get('page', 1);
         $offset = ($page-1)*self::ITEMS_PER_PAGE;
 
-        $countItems = $articleRepository->countItems();
+        $countItems = $articleRepository->countItemsByCategory(Article::CATEGORY_ARTICLE);
 
         return
             [
-                'articles' => $articleRepository->findList(self::ITEMS_PER_PAGE, $offset),
+                'articles' => $articleRepository->findListByCategory(self::ITEMS_PER_PAGE, $offset, Article::CATEGORY_ARTICLE),
```

Comment:
@daria-sieroshtan 
This does not apply here, but I remind you that following methods are preferrable:
```
$category->getArticles(); // you need corresponding method in entity, but it is generated by default
$articleRepository->findByCategory($category);  // Doctrine provides this via magic methods
```
Again, this is not applicable here, because neither method is paginatable, so just a reminder